test/check-networking

```diff
@@ -19,15 +19,35 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 from testlib import *
+import time
 import os
 
 class TestNetworking(MachineCase):
+    def setUp(self):
+        # we need a static ip for this test
+        MachineCase.setUp(self, macaddr='52:54:00:9e:00:F2')
 
     def get_iface(self, m, mac):
         def getit():
             return m.execute("for f in /sys/class/net/*; do a=$(cat $f/address); if test \"$a\" == %s; then echo $(basename $f); fi; done" % mac)
```

Comment:
I pushed a change to use grep instead of this. Should I put it on a separate branch?